specification/powerbidedicated/resource-manager/readme.md

```diff
@@ -0,0 +1,58 @@
+# PowerBIDedicated
+    
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for PowerBIDedicated.
+
+
+
+---
+## Getting Started 
+To build the SDK for PowerBIDedicated, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information 
+These are the global settings for the AnalysisServices API.
+
+``` yaml
+openapi-type: arm
+tag: package-2017-10-01
+```
+
+
+### Tag: package-2017-10-01
+
+These settings apply only when `--tag=package-2017-10-01` is specified on the command line.
+
+``` yaml $(tag) == package-2017-10-01'
+input-file:
+- Microsoft.PowerBIDedicated/package-2017-10-01/powerbidedicated.json
```

Comment:
realizing that this path looks incorrect, as I noticed a comment in the CI job about not finding the file in the config file. (https://travis-ci.org/Azure/azure-rest-api-specs/jobs/285536553#L617)
"package" is not part of the folder name where the api version is, path should be "Microsoft.PowerBIdedicated/2017-10-01/powerbidedicated.json", could you address this?

Reply:
Thank for catching that!
Fixed.